.github/workflows/jreleaser.yml

```diff
@@ -1,24 +1,16 @@
 name: Release
 
 on:
-  workflow_dispatch:
+  workflow_call:
```

Comment:
No, this is because jreleaser now is a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). The `release-manual` is the old `jreleaser`. I just wanted to combine them and not copy paste stuff three times.